package.json

```diff
@@ -14,6 +14,7 @@
     "start:app": "PORT=4141 react-scripts start",
```

Comment:
Thanks for setting this up @limdauto - I notice that your changes mentioned running viz from port 4142 while the proxy changes haven't been applied here yet - I assume you intend for this PR to be merged in after @bru5 's other PR?

Reply:
Yeah I'd agree I think add the proxy in this PR too as this appears to rely on that? Should be fine from a merge perspective.

Reply:
Yea proxy is already on `main`. Can either of you approve this PR please?

Reply:
sounds great, thanks both!